frontend/packages/knative-plugin/src/components/add/event-sources/SinkSection.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import * as fuzzy from 'fuzzysearch';
+import * as _ from 'lodash';
+import { useFormikContext, FormikValues } from 'formik';
+import { FormGroup } from '@patternfly/react-core';
+import { getFieldId } from '@console/shared';
+import { Dropdown, ResourceIcon } from '@console/internal/components/utils';
+import { referenceForModel, K8sKind, modelFor, referenceFor } from '@console/internal/module/k8s';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+
+type DropdownItemProps = {
+  model: K8sKind;
+  name: string;
+};
+
+const DropdownItem: React.FC<DropdownItemProps> = ({ model, name }) => (
+  <>
+    <span className={'co-resource-item'}>
```

Comment:
Is there not a reusable component for this already? You're relying on a lot of classes that are defined elsewhere.

```suggestion
    <span className="co-resource-item">
```